framework/util/const.go

```diff
@@ -1,15 +1,50 @@
 package util
 
+import (
+	"os"
+)
+
+var (
+	Envoy_Ratelimit             string
+	Envoy_Route                 string
+	Envoy_HttpConnectionManager string
+)
+
+func init() {
```

Comment:
also, how user can set this env while actually the deployment is managed by slimeboot.

Reply:
imporve the way of env passthrough